### PR TITLE
Update desktop notification browsers and convert to RST

### PR DESCRIPTION
**Pull request by esethna**
- Updated the browsers which desktop notifs are supported.
- Added note on desktop apps supporting desktop notifications
- converted to RST

### CODE REVIEW

File: source/help/getting-started/configuring-notifications.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Email Notifications
+-------------------------------------
+
+Emails are sent for any mentions you receive if you have Mattermost
+closed or have not had any browser activity for a short time.
+
+-  Turn email notifications On or Off in **Account Settings** >
+   **Notifications** > **Email Notifications**.
+-  Configure the email address where notifications are sent in **Account
```
Comment (lfbrock) [inline source/help/getting-started/configuring-notifications.rst]: Missing a period at the end of this bullet point

File: source/help/getting-started/configuring-notifications.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Mattermost is open. Desktop notifications are available on Edge, Firefox,
+Safari, Chrome and `Mattermost Desktop Apps <https://about.mattermost.com/download/#mattermostApps>`_.
+
+-  Configure when desktop notifications are sent from **Account
+   Settings** > **Notifications** > **Desktop Notifications** > **Send
+   Desktop Notifications**.
+-  Configure channel specific desktop notifications from **Channel
+   Menu** > **Notification Preferences** > **Send Desktop
+   Notifications**. By default, all channels use the global setting
+   configured in *Account Settings*.
```
Comment (lfbrock) [inline source/help/getting-started/configuring-notifications.rst]: Do we have a guideline for using italics here? If not maybe queue for discussion?

File: source/help/getting-started/configuring-notifications.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+   push notifications**.
+-  Configure when push notifications are sent depending on your status
+   from **Account Settings** > **Notifications** > **Mobile Push
+   Notifications** > **Trigger push notifications when**.
+
+Browser Tab Notifications
+----------------------------------------
+
+The browser tab favicon updates to inform you of unread messages and
+mentions if Mattermost is open in an inactive browser tab. Browser tab
```
Comment (lfbrock) [inline source/help/getting-started/configuring-notifications.rst]: Does this need to be an inactive browser tab?

File: source/help/getting-started/configuring-notifications.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+You can choose not to show unread indicators in a channel by clicking
+**Channel Menu** > **Notification Preferences** > **Mark Channel
+Unread** and selecting **Only for mentions**. This will only notify you if a mention is triggered on your name or a
+keyword for which you're listening.
+
+Mentions Indicator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unread mentions or `your name and selected
```
Comment (lfbrock) [inline source/help/getting-started/configuring-notifications.rst]: This sentence is a little confusing, are there any ways we could phrase it better? Also should the docs link be for the words "your name and selected keywords" or just "selected keywords"?

File: source/help/getting-started/configuring-notifications.rst (hunk 2 above)
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: Yes we decided that Italics are for windows or sections that guide the user but not for clicks.

File: source/help/getting-started/configuring-notifications.rst (hunk 1 above)
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: fixed

**Comment by esethna** [comment]
Updated based on feedback from LB.

**Comment by lfbrock** [comment]
+1

**Pull request merged**